Fix hard-coded absolute paths in fonts{2,3}go.cgi

tim-eves opened the pull request:
The paths as checked in are incorrect for the server and relative paths will work just as well and provide more future flexibility.

tim-eves committed:
Update fonts2go.cgi

tim-eves committed:
Fix hardcoded path to TypeTuner

Make relative to the script so we don't need to fixup the hardcoded path on checkout

LornaSIL commented:
@tim-eves I'm seeing the Awami Nastaliq 1.900 font now at fonts3go, but when I select the font and click on "Select Features" I get this message:

```
Software error:

External program exited with return code 2
Can't open font at typetuner.pl line 1076.

For help, please send mail to this site's webmaster, giving this error message and the time and date of the error. 
```

bobh0303 commented:
Same error code for fonts2go

tim-eves committed:
Fix the fix

The base path is relative to the script but it needs to resolved to an absolute dir first or inner modules will treat it as a relative path too

tim-eves committed:
Fix the fix for fonts2go.cgi

The base path is relative to the script but it needs to resolved to an absolute dir first or inner modules will treat it as a relative path too

tim-eves commented:
Should be fixed for both now
